students/ISMAEL/style.css

```diff
@@ -0,0 +1,129 @@
+body {
```

Comment:
Try to be consistent with how many spaces you use between each rule. There are parts where you use 1 and parts where you use none. I would use always 1. There are some class names that you could try to find a more readable and semantically way to describe them like the class .h2, or the class .h22.